meta-xt-driver-domain/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -0,0 +1,11 @@
+
+IMAGE_INSTALL_append = " \
```

Comment:
I can see that this is a copy of core-image-weston.bbappend.

I believe it is better to create .inc file with this contents and include it from both .bbappends